admin/src/components/header/invite.js

```diff
@@ -173,6 +198,65 @@ const ChooseRegion = ({ value, onChange }) => {
   );
 };
 
+const ChooseCenter = ({ value, onChange, centers, onSelect }) => {
+  const { user } = useSelector((state) => state.Auth);
+
+  // useEffect(() => {
+  //   if (user.role === REFERENT_ROLES.HEAD_CENTER) {
+  //     return (
+  //       onChange({ target: { value: user.cohesionCenterId, name: "cohesionCenterId" } }),
+  //       onChange({ target: { value: user.cohesionCenterName, name: "cohesionCenterName" } })
+  //     )
+  //   }
+  //   return (
+  //     (e) => {
+  //       console.log('VALUE', JSON.stringify(e));
+  //       onChange({ target: { value: e._id, name: "cohesionCenterId" } });
+  //       onChange({ target: { value: e.value, name: "cohesionCenterName" } });
+  //       onSelect?.(e);
+  //     }
+  //   )
+  // }, []);
+
+  return (
+    // <Input
+    //   disabled={user.role === REFERENT_ROLES.HEAD_CENTER}
+    //   type="select" name="cohesionCenterName"
+    //   value={value}
+    //   onChange={(e) => {
+    //     const name = e.target.value
+    //     const _id = e.target.key
+    //     console.log('--VALUE--', name, _id, e);
+    //     onChange({ target: { value: name, name: "cohesionCenterId" } });
+    //     onChange({ target: { value: _id, name: "cohesionCenterName" } });
+    //   }}
+    // >
+    //   {centers?.map((e) => {
+    //     return (
+    //       <option value={e.name} key={e._id}>
+    //         {e.name}
+    //       </option>
+    //     );
+    //   })}
+    // </Input>
+    <ReactSelect
+      options={centers}
```

Comment:
je crois qu'on n'a pas autorisé les chefs de centre a inviter d'autres chef de centre, mais autant le mettre pour une possible futur amélioration, ca mange pas de pain
```suggestion
      disabled={user.role === REFERENT_ROLES.HEAD_CENTER}
      options={centers}
```